pkg/component/etcd/monitoring.go

```diff
@@ -174,51 +174,51 @@ const (
   # etcd backup failure alerts
   - alert: KubeEtcdDeltaBackupFailed
     expr:
-        (
             (
-                time() - ` + monitoringMetricBackupRestoreSnapshotLatestTimestamp + `{job="` + monitoringPrometheusJobBackupRestoreNamePrefix + `-{{ .role }}",kind="Incr"}
-              > bool
-                900
+                (
+                    time() - ` + monitoringMetricBackupRestoreSnapshotLatestTimestamp + `{job="` + monitoringPrometheusJobBackupRestoreNamePrefix + `-{{ .role }}",kind="Incr"}
+                  > bool
+                    900
+                )
+              *
+                etcdbr_snapshot_required{job="` + monitoringPrometheusJobBackupRestoreNamePrefix + `-{{ .role }}",kind="Incr"}
             )
-          +
-            (etcdbr_snapshot_required{job="` + monitoringPrometheusJobBackupRestoreNamePrefix + `-{{ .role }}",kind="Incr"} >= bool 1)
-          ==
-            2
-        )
-      + on (pod, role) group_left ()
-        0 * (` + monitoringMetricEtcdServerIsLeader + `{job="` + monitoringPrometheusJobEtcdNamePrefix + `-{{ .role }}"} == 1)
+          * on (pod, role)
```

Comment:
Here you are no longer using the `group_left` modifier (see https://prometheus.io/docs/prometheus/latest/querying/operators/#many-to-one-and-one-to-many-vector-matches)...